packs/salt/actions/lib/base.py

```diff
@@ -1,9 +1,23 @@
+# pylint: disable=no-member
+
 from st2actions.runners.pythonrunner import Action
 from requests import Request
+import logging
```

Comment:
Yeah. I was following the logging approach used in the elasticsearch pack. I can switch it. Just say the word. :)